File: 12-math/A.go
package main

import (
	"fmt"
)

func gcd(a, b int) int {
	for b != 0 {
		a, b = b, a%b
	}
	return a
}

func lcm(a, b int) int {
	return (a * b) / gcd(a, b)
}

func main() {
	var N, K int
	fmt.Scanf("%d %d", &N, &K)

	lcmValue := lcm(N, K)

	fmt.Println(lcmValue)
}

File: 12-math/A.cs
using System;

class Program
{
    static int Gcd(int a, int b)
    {
        while (b != 0)
        {
            int temp = b;
            b = a % b;
            a = temp;
        }
        return a;
    }

    static int Lcm(int a, int b)
    {
        return (a * b) / Gcd(a, b);
    }

    static void Main()
    {
        string input = Console.ReadLine();
        string[] values = input.Split();
        int N = int.Parse(values[0]);
        int K = int.Parse(values[1]);

        int lcmValue = Lcm(N, K);

        Console.WriteLine(lcmValue);
    }
}